ci/prow/config_knative.yaml

```diff
@@ -254,3 +260,13 @@ periodics:
   knative/observability:
     - continuous: true
       cron: "30 * * * *" # Run every hour and 30 minutes
+
+  GoogleCloudPlatform/cloud-run-events:
```

Comment:
```suggestion
  googlecloudplatform/cloud-run-events:
```

Reply:
It is CamelCased.

Reply:
https://github.com/GoogleCloudPlatform/cloud-run-events

Reply:
Yes, I saw that. But it messes up with all the buckets and tabs and names generated from it. I don't think it will break fetching repos; but if it does I'll revert it.

Reply:
Got it. Fixed.